R/module_teal_data.R

```diff
@@ -37,7 +37,10 @@
 NULL
 
 #' @rdname module_teal_data
-ui_teal_data <- function(id, data_module = function(id) NULL) {
+#' @aliases ui_teal_data
+#' @note
+#' `ui_teal_data_module` was renamed from `ui_teal_data`.
+ui_teal_data_module <- function(id, data_module = function(id) NULL) {
```

Comment:
Is `ui_teal_data` used in other teal packages? Or is it internal `teal` function?

Reply:
I searched via `grep` ui_teal_data and srv_teal_data and I didn't found any usage in other packages (and just a single call to srv_teal_data in teal).  They are marked as internal too on R/module_teal_data.R L36. 

I didn't modify the exported/imported status. Yes, it is internal and not exported anymore.